dvc/command/config.py

```diff
@@ -29,6 +29,15 @@ def run(self):
                 section, opt, self.args.value, level=self.args.level
             )
 
+        is_write = self.args.unset or self.args.value is not None
+        if is_write and self.args.name == "cache.type":
+            logger.warning(
+                "You have changed cache.type, this doesn't update link types "
```

Comment:
cc @jorgeorpinel

Reply:
You have changed the \`cache.type\` option.
This doesn't update existing file links/copies in
the workspace, but it can be done with:

	dvc checkout --relink

I'm not sure this formatting with 12 spaces before the command is consistent with other dvc output, like when you `dvc add` and it gives you the exact `git add` command, in which case it seems to use a tab`\t`  instead.

Reply:
Without those spaces it looks really weird on wider screen:
```
WARNING: You have changed the 'cache.type' option ...
    dvc checkout --relink
```
Since we don't have any policy about it, let's keep it this way.

Reply:
I see this you used most of my suggestion, thanks. Also this is merged and I don't know if there's an output indentation policy but my 2c here is that the default should be to keep things consistent with existing instances, and I see mainly tabs in existing output (not sure though). Up to you guys cc @efiop

Reply:
@jorgeorpinel Good point. We don't have any set style right now, so I would leave it as is for now and just keep the thought in my head to unify this later. Thanks for the heads up! 🙂